cmd/collect.go

```diff
@@ -0,0 +1,88 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/prometheus/client_golang/api"
+	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+
+	"github.com/numaproj-labs/numaflow-perfman/collect"
+	"github.com/numaproj-labs/numaflow-perfman/util"
+)
+
+var (
+	Name    string
+	Last    int
+	Metrics []string
+)
+
+var validMetrics []string
+var metricsMatrix = map[string][]collect.MetricObject{
+	"data-forward": {
```

Comment:
The name `data-forward` could be confusing. Use metric-related names like `latency`, `throughput`, `error` etc.